### PR TITLE
add mock vc sysfs

### PR DESCRIPTION
**Pull request by anandaravuri**

### CODE REVIEW

File: framework/platform/fpga_hw.cpp [hunk 1]
```diff
@@ -613,7 +639,8 @@ static std::map<platform_cfg, std::string> platform_names = {
   {  platform_cfg(0x8086, 0x09c4, fpga_driver::linux_dfl0), "dcp-rc-dfl0_patchset2" },
   {  platform_cfg(0x8086, 0xbcc0, fpga_driver::linux_dfl0),  "skx-p-dfl0" },
   {  platform_cfg(0x8086, 0x0b30, fpga_driver::linux_intel), "dcp-vc" },
-  {  platform_cfg(0x8086, 0x0b31, fpga_driver::linux_intel), "dcp-vc-v" }
+  {  platform_cfg(0x8086, 0x0b31, fpga_driver::linux_intel), "dcp-vc-v" },
```
Comment (tswhison) [inline framework/platform/fpga_hw.cpp]: Is dcp-vc-v the correct name?

Comment (anandaravuri) [reply framework/platform/fpga_hw.cpp]: removed Fixed

File: framework/platform/fpga_hw.cpp [hunk 2]
```diff
@@ -686,7 +712,8 @@ std::pair<std::string, test_platform> make_platform(uint16_t ven_id, uint16_t de
   platform.mock_sysfs = nullptr;
   std::string name = platform_names[platform_cfg(ven_id, dev_id, platform.driver)];
   for (auto p : pci_paths) {
-    platform.devices.push_back(make_device(ven_id, dev_id, name, p));
+	if (!name.empty())
```
Comment (tswhison) [inline framework/platform/fpga_hw.cpp]: Why is this required?

Comment (anandaravuri) [reply framework/platform/fpga_hw.cpp]: removed if condition
